.circleci/config.yml

```diff
@@ -10,12 +10,8 @@ jobs:
       - image: microsoft/dotnet:2.0-sdk-jessie
       - image: redis
     steps:
-      - run:
-          name: install packages
-          command: apt-get -q update && apt-get install -qy awscli
       - checkout
-      - run: aws s3 cp s3://launchdarkly-pastebin/ci/dotnet/LaunchDarkly.Redis.snk LaunchDarkly.Redis.snk
```

Comment:
I've eliminated the downloading of the key file as I did in dotnet-client, because the Debug build now skips the signing step.